src/views/Pools/components/CardFooter.tsx

```diff
@@ -116,46 +126,47 @@ const CardFooter: React.FC<Props> = ({
         </StyledDetailsButton>
       </Row>
       {isOpen && (
-        <Details>
-          <Row mb="4px">
+        <>
+          <Row mb="4px" mt="20px">
             <FlexFull>
-              <Label>
-                <span role="img" aria-label="syrup">
-                  🥞{' '}
-                </span>
-                {TranslateString(408, 'Total')}
-              </Label>
+              <Text fontSize="14px">{TranslateString(999, 'Total staked')}</Text>
             </FlexFull>
-            <Balance fontSize="14px" isDisabled={isFinished} value={getBalanceNumber(totalStaked, decimals)} />
+            <Balance fontSize="14px" value={getBalanceNumber(totalStaked, decimals)} bold={false} />
+            &nbsp;
+            <Text fontSize="14px">{tokenName}</Text>
           </Row>
           {blocksUntilStart > 0 && (
-            <Row mb="4px">
+            <Row>
               <FlexFull>
-                <Label>{TranslateString(410, 'Start')}:</Label>
+                <Text fontSize="14px">{TranslateString(410, 'Start')}:</Text>
               </FlexFull>
               <Balance fontSize="14px" isDisabled={isFinished} value={blocksUntilStart} decimals={0} />
             </Row>
           )}
           {blocksUntilStart === 0 && blocksRemaining > 0 && (
-            <Row mb="4px">
+            <Row>
               <FlexFull>
-                <Label>{TranslateString(410, 'End')}:</Label>
+                <Text fontSize="14px">{TranslateString(410, 'End')}:</Text>
               </FlexFull>
-              <Balance fontSize="14px" isDisabled={isFinished} value={blocksRemaining} decimals={0} />
+              <Balance fontSize="14px" value={blocksRemaining} decimals={0} color="primary" bold={false} />
+              <TimerIcon color="primary" width="16px" />
             </Row>
           )}
-          {tokenAddress && (
+          <StyledLinkExternal href={projectLink} ml="auto">
+            {TranslateString(999, 'Project site')}
+          </StyledLinkExternal>
+          <StyledLinkExternal href="#" ml="auto">
+            {TranslateString(999, 'Info site')}
+          </StyledLinkExternal>
+          {!!account && (
             <Flex mb="4px">
               <TokenLink onClick={() => registerToken(tokenAddress, tokenName, tokenDecimals, imageSrc)}>
                 Add {tokenName} to Metamask
               </TokenLink>
```

Comment:
This is incorrect, it should be the earning token being added to metamask, not the staking token.